google-cloud-clients/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/SequentialExecutorService.java

```diff
@@ -146,29 +147,66 @@ private void invokeCallbackAndExecuteNext(final String key, final Deque<Runnable
       super(executor);
     }
 
+    /**
+     * This method does the following in a chain:
+     *
+     * <ol>
+     *   <li>Creates an `ApiFuture` that can be used for tracking progress.
+     *   <li>Creates a `CancellableRunnable` out of the `Callable`
+     *   <li>Adds the `CancellableRunnable` to the task queue
+     *   <li>Once the task is ready to be run, it will execute the `Callable`
+     *   <li>When the callable completes one of two things happens:
```

Comment:
Write `Callable` consistently (instead of callable), i.e.:
When the `Callable` completes one of two things happens: